src/lib/get-graphql-rate-limiter.ts

```diff
@@ -128,7 +137,10 @@ const getGraphQLRateLimiter = (
     });
 
     // Fetch timestamps from previous requests out of the store.
-    const accessTimestamps = await store.getForIdentity(identity);
+    const accessTimestamps = await store.getForIdentity(identity).catch(e => {
```

Comment:
Have you considered just using synchronous code synchronously and resort to try catch?